test/Microsoft.ML.StaticPipelineTesting/Microsoft.ML.StaticPipelineTesting.csproj

```diff
@@ -6,15 +6,15 @@
     <ProjectReference Include="..\..\src\Microsoft.ML.Data\Microsoft.ML.Data.csproj" />
     <ProjectReference Include="..\..\src\Microsoft.ML.ImageAnalytics\Microsoft.ML.ImageAnalytics.csproj" />
     <ProjectReference Include="..\..\src\Microsoft.ML.StandardLearners\Microsoft.ML.StandardLearners.csproj" />
-    <ProjectReference Include="..\Microsoft.ML.TestFramework\Microsoft.ML.TestFramework.csproj" />
-    <NativeAssemblyReference Include="FactorizationMachineNative" />
-
+    <ProjectReference Include="..\Microsoft.ML.TestFramework\Microsoft.ML.TestFramework.csproj" />    
     <ProjectReference Include="..\..\src\Microsoft.ML.Analyzer\Microsoft.ML.Analyzer.csproj">
       <ReferenceOutputAssembly>false</ReferenceOutputAssembly>
       <OutputItemType>Analyzer</OutputItemType>
     </ProjectReference>
 
     <NativeAssemblyReference Include="CpuMathNative" />
-    <NativeAssemblyReference Include="LdaNative" />
+    <NativeAssemblyReference Include="FactorizationMachineNative" />
+    <NativeAssemblyReference Include="FastTreeNative" />
+	<NativeAssemblyReference Include="LdaNative" />
```

Comment:
>	 [](start = 0, length = 1)

tab

Reply:
Tools->Options->Text Editor->All Languages->Insert Spaces

Checked!

---
In reply to: [220301837](https://github.com/dotnet/machinelearning/pull/1009#discussion_r220301837) [](ancestors = 220301837)